library/LuaApi.cpp

```diff
@@ -1621,6 +1634,7 @@ static const luaL_Reg dfhack_gui_funcs[] = {
     { "pauseRecenter", gui_pauseRecenter },
     { "revealInDwarfmodeMap", gui_revealInDwarfmodeMap },
     { "getMousePos", gui_getMousePos },
+    { "getFocusString", gui_getFocusString },
```

Comment:
you might as well rename this to `getFocusStrings` since we're changing the return value anyway.